index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,700;1,400&display=swap"
+      rel="stylesheet"
+    />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio Project</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <section>
+      <header class="header-toolbar">
+        <a href="#" id="my-logo">
+          <h2 id="logo">My Logo</h2>
+        </a>
+        <nav id="vector-1">
+          <button>
+            <a href=""
+              ><img
+                src="./images/Vector button.png.jpeg"
+                alt="vector-button"
+                class="link"
+            /></a>
+          </button>
+        </nav>
+      </header>
+      <section class="headline-section">
+        <div class="headline-container">
+          <h1 id="header-text">
+            I'm Brian.<br />
+            Glad to see <br />
+            you!
+          </h1>
+          <p id="headline-paragraph">
+            I'm a software developer! I can help you build a product, feature or
+            website look through some of my work and experience! if you like
+            what you see and have a project you need coded, dont hesitate to
+            contact me.
+          </p>
+          <a
+            href="https://www.linkedin.com/feed/"
+            id="contact-link"
+            class="link"
+          >
+            LET'S CONNECT</a
+          >
+          <p class="contact-icons">
+            <a href="https://twitter.com/bk_rian"
+              ><img src="./images/twitter.svg" alt="twitter-icon"
+            /></a>
+            <a href="https://www.linkedin.com/feed/"
+              ><img src="./images/in.png.jpeg" alt="linked-icon" />
+            </a>
+            <a href="https://github.com/Brian3Kamau"
+              ><img src="./images/M.png.jpeg" alt="M-icon" />
+            </a>
+            <a href="https://github.com/Brian3Kamau"
+              ><img src="./images/github.png.jpeg" alt="github-icon" />
+            </a>
+            <a href=""><img src="" alt="" /> </a>
+            <a href="https://www.google.com"
+              ><img src="./images/fing.png.jpeg" alt="fng-icon" />
+            </a>
+          </p>
```

Comment:
- [ ] [OPTIONAL] I would strongly suggest to inclose your icons in to a `ul` and `li` elements instead of just paragraph tag `p` for semantic purpose. When the browser reads your HTML, it interprets all the tags that surround your text. The tags tell the browser about the structure and meaning of your text. The `<p>` tag defines a paragraph, whereas` ul` `li` explicitly mean "this is an un/ordered list of items." The term "semantics" refers to the way that you use the inherent meaning of existing structures to create explicit meaning.